client/my-sites/checkout/upsell-nudge/professional-email-upsell/index.tsx

```diff
@@ -17,30 +17,43 @@ import {
 	transformMailboxForCart,
 	validateMailboxes,
 } from 'calypso/lib/titan/new-mailbox';
+import type { MinimalRequestCartProduct } from '@automattic/shopping-cart';
+import type { TranslateResult } from 'i18n-calypso';
+import type { ChangeEvent } from 'react';
 
 import './style.scss';
 
-const noopWithCallback = ( ignored, callback = () => {} ) => {
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noopWithCallback = ( cartItem: MinimalRequestCartProduct, callback = () => {} ) => {
```

Comment:
Super NIT: 

why not define `const noop = () => {};` as in other places (as for example here: [client/my-sites/checkout/upsell-nudge/index.jsx](https://github.com/Automattic/wp-calypso/pull/60465/files#R73)?)

Reply:
The function needs to accept a callback as the second option and needs to call the supplied callback. But it may not be necessary to have this function defined at all - we can simply require that the caller specify it.

Reply:
I've removed the default function and just require the function to be supplied.